modules/shared/src/test/scala/org/tessellation/block/generators.scala

```diff
@@ -19,7 +19,7 @@ object generators {
     for {
       blockReferences <- blockReferencesGen
       signedTxn <- signedTransactionGen
-    } yield Block(blockReferences, NonEmptySet.fromSetUnsafe(SortedSet(signedTxn)))
+    } yield Block(blockReferences, NonEmptySet.fromSetUnsafe(SortedSet(signedTxn)), None)
```

Comment:
```suggestion
    } yield Block(blockReferences, NonEmptySet.one(signedTxn), None)
```

Reply:
It's not part of my changes but I will change it as you suggested :+1: